spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -335,6 +335,7 @@ object TableFeature {
       IcebergCompatV1TableFeature,
       IcebergCompatV2TableFeature,
       DeletionVectorsTableFeature,
+      VacuumProtocolCheckTableFeature,
```

Comment:
As part of this change, we should also finalize the vacuumProtocolCheck RFC and update the Delta spec.

Reply:
Added it in a separate PR https://github.com/delta-io/delta/pull/2808/files